### PR TITLE
Backgroundimage

### PR DESCRIPTION
**Pull request by thieme55**
gradient background with svg image which scales resposivly to screen size
-styl.css background stuff
to make other sites the same just copie the two div's from landingpage.html (these are markt)

In addition: background was added to the loading logo-page animation.

### CODE REVIEW

File: server/swissvoice/static/css/style.css [hunk 1]
```diff
@@ -81,3 +112,15 @@ hr.hz-line {
         font-size: 1.2rem;
     }
 }
+
+
+/*--------------------------------------------------*/
+/*medis querry for background-image*/
```
Comment (siku2) [inline server/swissvoice/static/css/style.css]: media query?

File: server/swissvoice/static/css/landingpage.css [hunk 2]
```diff
@@ -10,15 +10,20 @@
...
 }
 
 #main-schreiben-btn {
   font-size: 1.5rem;
-  background-color:  #007fff; /*color of button in action-section*/
+  background-color:  #63aeff; /*color of button in action-section*/
   border-color: transparent;
 }
 
+#main-schreiben-btn:hover {
```
Comment (siku2) [inline server/swissvoice/static/css/landingpage.css:22]: why the denglish?

File: server/swissvoice/static/landingpage.html [hunk 3]
```diff
@@ -10,8 +10,14 @@
     <script src="js/landing-page.bundle.js"></script>
 </head>
 <body>
+
+  <!--background image and gradien -->
```
Comment (siku2) [inline server/swissvoice/static/landingpage.html:14]: gradient

File: server/swissvoice/static/css/loading-screen.css [hunk 4]
```diff
@@ -1,5 +1,26 @@
 /*here is the CSS !only! for the loading animation*/
 
+.bg-logo-screen {
```
[the remark this answers is not in the document]
Comment (siku2) [reply server/swissvoice/static/css/loading-screen.css]: They work like this:
```css
background: -moz-linear-gradient(top,  #1e5799 0%, #7db9e8 100%); /* FF3.6+ */
background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,#1e5799), color-stop(100%,#7db9e8)); /* Chrome,Safari4+ */
background: -webkit-linear-gradient(top,  #1e5799 0%,#7db9e8 100%); /* Chrome10+,Safari5.1+ */
```

File: server/swissvoice/static/css/style.css [hunk 5]
```diff
@@ -4,6 +4,34 @@ body {
...
+  bottom: -50vh;
+  right: 0;
+  width: 50rem;
+  height: 60rem;
+/*image modifications*/
+  opacity: 0.2;
+
+}
+
+#bg-gradient {
```
[the remark this answers is not in the document]
Comment (siku2) [reply server/swissvoice/static/css/style.css]: refer to my comment above

File: server/swissvoice/static/css/style.css [hunk 6]
```diff
@@ -4,6 +4,34 @@ body {
     padding: 0;
 }
 
+#bg-bubble {
+  background-image: url(../images/background/sprechblase.svg);
```
Comment (siku2) [inline server/swissvoice/static/css/style.css]: while I do appreciate a good relative url, I think that in this case we should use `/images/background/sprechblase.svg` (starting with a `/` means that the url is relative to the domain)

File: server/swissvoice/static/css/style.css [hunk 7]
```diff
@@ -4,6 +4,34 @@ body {
...
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  position: fixed;
+/* positioning*/
+  bottom: -50vh;
+  right: 0;
+  width: 50rem;
```
Comment (siku2) [inline server/swissvoice/static/css/style.css:15]: is there a good reason to use "rem" here?

File: server/swissvoice/static/css/style.css (hunk 6 above)
Comment (siku2) [reply server/swissvoice/static/css/style.css]: Also, using a German filename all of a sudden doesn't really make sense.

**Review by siku2 — CHANGES_REQUESTED** [review]

File: server/swissvoice/static/css/landingpage.css (hunk 2 above)
Comment (thieme55) [reply server/swissvoice/static/css/landingpage.css:22]: Caus this button will (in the futur) lead to a page were only the "schreiben" will take place.

File: server/swissvoice/static/css/style.css (hunk 6 above)
Comment (thieme55) [reply server/swissvoice/static/css/style.css]: Didn't really know how to url
german filename will be changed

File: server/swissvoice/static/css/style.css (hunk 7 above)
Comment (thieme55) [reply server/swissvoice/static/css/style.css:15]: Using rem so the img is better controlable and dosen't stretch wierdly when making it responsiv

File: server/swissvoice/static/css/style.css (hunk 1 above)
Comment (thieme55) [reply server/swissvoice/static/css/style.css]: Is needed so the bubble doesn't cover up the titel (only the case for smaller screens and height dependent)

**Review by siku2 — APPROVED** [review]

**Pull request merged**